CoreScriptsRoot/Modules/VR/Recenter.lua

```diff
@@ -1,10 +1,13 @@
 local InputService = game:GetService("UserInputService")
+local ContextActionService = game:GetService("ContextActionService")
 local CoreGui = game:GetService("CoreGui")
 local RobloxGui = CoreGui:WaitForChild("RobloxGui")
 local Panel3D = require(RobloxGui.Modules.VR.Panel3D)
 local VRHub = require(RobloxGui.Modules.VR.VRHub)
 local Util = require(RobloxGui.Modules.Settings.Utility)
 
+local cancelShortcutName = game:GetService("HttpService"):GenerateGUID()
```

Comment:
Since we are a core action we don't need to create a GUID and a comprehensive name may be better